### PR TITLE
Initial database models for player and scene

### PR DESCRIPTION
**Pull request by nmughal**
This is the initial setup for the database (with model and schema for player and scene) and includes the functions to add a player and a scene to the database.

### CODE REVIEW

File: server/routes/player.routes.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+function addAPlayer(request, response, next) {
+  console.log('Incoming', request.body);
+
+  if(!request.body) {
+    let err = new Error('You must provide a player');
+    err.status = 400;
+    next(err);
+    return;
+  }
+
```
Comment (davidasteed) [inline server/routes/player.routes.js]: we'll need at some point validation here to fail if 
a) name or email address property is missing or blank
b) whether a player with the same email address exists

...so at the very least we need a comment to remind ourselves or at least, a User Story to handle data validation

File: server/routes/scene.routes.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  let theSceneCreated = new Scene({
+    sceneImage: request.body.sceneImage,
+    sceneText: request.body.sceneText,
+    sceneChoice: {
+      choiceText: request.body.sceneChoice.choiceText,
+      choiceScore: request.body.sceneChoice.choiceScore
+    }
+  });
+  console.log(theSceneCreated);
```
Comment (davidasteed) [inline server/routes/scene.routes.js]: remove unneeded console log statements, or leave them since I need to validate when I code the gameplay page?

**Comment by nmughal** [comment]
Created the data audits for add functions and error handler middleware.

File: package.json [hunk 3]
```diff
@@ -2,11 +2,14 @@
   "name": "iron-adventure-game",
   "version": "1.0.0",
   "description": "Adventure game \"capstone project\" for The Iron Yard Feb 2017 cohort",
-  "main": "",
+  "main": "server/app.js",
   "dependencies": {
     "angular": "^1.6.4",
     "angular-ui-router": "^0.4.2",
```
Comment (davidasteed) [inline package.json]: for some reason, you have two packages named "body" and "parser":  I have a single one:

  "dependencies": {
    "body-parser": "^1.17.1",

**Pull request merged**